awslambda/cml/cml_manager.py

```diff
@@ -92,80 +73,84 @@ def manage_labs(self) -> tuple:
+                    labs_warning_stopped.append(
+                        (lab_id, lab_title, user_responded_date)
+                    )
 
                 # check see if within deletion period
                 elif self.lab_to_delete(
-                    lab_wiped_date, user_responded_date, lab_is_wiped
+                    lab_stopped_date, user_responded_date, lab_is_stopped
                 ):
                     self.logging.info("Adding lab to be deleted")
-                    labs_to_delete.append(lab_id)
+                    labs_to_delete.append({"lab_id": lab_id, "user_email": user_email})
 
-                # check see if lab within wiped period
+                # check see if lab within stopped period
                 elif self.lab_to_warn_delete(
-                    lab_wiped_date,
+                    lab_stopped_date,
                     user_responded_date,
-                    lab_is_wiped,
+                    lab_is_stopped,
                     lab_id,
                     lab_title,
                     user_email,
                 ):
-                    self.logging.info("Adding lab to warning for being wiped")
-                    labs_warning_deleted.append((lab_title, lab_wiped_date))
+                    self.logging.info("Adding lab to warning for being stopped")
+                    labs_warning_deleted.append((lab_title, lab_stopped_date))
 
-                # Checks to see if a wiped lab has been reactivated
-                elif lab_is_wiped and self.cml_api.check_lab_active(lab_id):
+                # Checks to see if a stopped lab has been reactivated
+                elif lab_is_stopped and self.cml_api.check_lab_active(lab_id):
                     self.dynamodb.update_cml_lab_used_date(
                         user_email, lab_id, lab_title
                     )
+                    self.logging.info("Lab reset %s", lab_title)
 
-            self.logging.info("WARN WIPE: %s", str(labs_warning_wiped))
-            self.logging.info("WIPE: %s", str(labs_to_wipe))
-            self.logging.info("WARN DELETE: %s", str(labs_warning_deleted))
-            self.logging.info("DELETE: %s", str(labs_to_delete))
-
-            # Delete labs
-            self.cml_api.delete_labs(labs_to_delete, user_email)
+            # Stop labs
+            self.cml_api.stop_labs(labs_to_stop, user_email)
 
-            # Wipe labs
-            self.cml_api.wipe_labs(labs_to_wipe, user_email)
-
-            # Send card warning labs to be wiped
-            self.send_labbing_card(labs_warning_wiped, user_email)
+            # Send card warning labs to be stopped
+            self.send_labbing_card(labs_warning_stopped, user_email)
 
             # Send card warning labs to be deleted
             self.send_deletion_card(labs_warning_deleted, user_email)
 
+            # Add Labs to delete
+            all_labs_to_delete += labs_to_delete
+
+        # Delete labs
+        self.cml_api.start_delete_process(all_labs_to_delete)
+
```

Comment:
not part of this PR, but i noticed that success_counter/fail_counter are never updated